scripts/update/Updater.php

```diff
@@ -407,5 +407,10 @@ public function update($currentVersion)
             call_user_func(new RegisterPciAudioRecording(), ['0.10.0']);
             $this->setVersion('6.2.0');
         }
+
+        if($this->isVersion('6.2.0')){
+            call_user_func(new RegisterPciMathEntry(), []);
```

Comment:
You migth need to specify the version number here?
`call_user_func(new RegisterPciMathEntry(), ['0.9.1']);`